checkouts.cfg

```diff
@@ -11,13 +11,36 @@ auto-checkout =
 # manual added
 # automatically added by mr.roboto
 # Unfortunately does not happen anymore, see https://github.com/plone/mr.roboto/issues/79
-    mockup
-    plone.app.content
     Products.CMFCore
     Products.CMFEditions
-    plone.recipe.zope2instance
     Products.CMFPlacefulWorkflow
-    plone.app.discussion
     Products.PlonePAS
+    mockup
     plone.app.caching
+    plone.app.content
+    plone.app.contentmenu
+    plone.app.contentrules
+    plone.app.contenttypes
+    plone.app.dexterity
+    plone.app.discussion
+    plone.app.event
+    plone.app.layout
+    plone.app.mosaic
```

Comment:
Note the build now fails with: `The package 'plone.app.mosaic' from auto-checkout has no source information.`

Reply:
ouch ... sorry, p.a.mosaic should not be there